config/software/redis.rb

```diff
@@ -63,10 +50,8 @@
 
   update_config_guess
 
-  if version.satisfies?(">= 3.0.7", "< 3.1")
+  if version.satisfies?("< 6.0")
     patch source: "password-from-environment.patch", plevel: 1, env: env
-  elsif version.satisfies?(">= 5.0", "< 6.0")
-    patch source: "password-from-environment-5.patch", plevel: 1, env: env
```

Comment:
don't you want to retain the -5 patch and drop the older one for 3.x?

Reply:
I renamed the patch so it looks like we're using the 3 patch here, but it's the 5 patch